iml-orm/src/models.rs

```diff
@@ -483,6 +484,24 @@ pub struct ChromaCoreDetecttargetsjob {
     pub host_ids: String,
 }
 
+#[derive(serde::Serialize, serde::Deserialize)]
+#[cfg_attr(feature = "postgres-interop", derive(Queryable, Debug, Identifiable))]
+#[cfg_attr(feature = "postgres-interop", primary_key(id))]
+#[cfg_attr(feature = "postgres-interop", table_name = "chroma_core_device")]
+pub struct ChromaCoreDevice {
+    pub id: i32,
+    pub fqdn: String,
+    pub device: serde_json::Value,
```

Comment:
```suggestion
    pub devices: serde_json::Value,
```

Since this is a DAG of devices.